tasks/configure_logrotate.yml

```diff
@@ -0,0 +1,12 @@
+---
+- name: Get Vertica node name
+  command: /opt/vertica/bin/vsql -U dbadmin -w {{ vertica_dbadmin_password }} -t -A -c "select NODE_NAME from NODES;"
+  sudo_user: dbadmin
+  environment: "{{vertica_env}}"
+  register: vertica_node
+
+- name: Set vertica_node name
+  set_fact: vertica_node_name={{vertica_node.stdout_lines}}
+
+- name: Place logrotate conf for vertica
+  template: src=vertica_logrotate.j2 dest=/opt/vertica/config/logrotate/mon owner=root group=root mode=644
```

Comment:
Could we make the database name a parameter? Have it set to mon by default, but allow it to be overridden?